plugins/generator-1.16.4/forge-1.16.4/procedures/get_localized_text.java.ftl

```diff
@@ -1 +1 @@
-(new TranslationTextComponent(${input$key}).getUnformattedComponentText())
\ No newline at end of file
+(new TranslationTextComponent(${input$key}).toString())
```

Comment:
What does this fix?

Reply:
Well with the old one, the text just wouldn't appear. You're welcome to test with the old version and with my new one.